creator-node/scripts/register.js

```diff
@@ -19,7 +19,7 @@ async function main() {
       config.get("ethTokenAddress"),
       config.get("ethRegistryAddress"),
       ethWeb3,
-      config.get("ethOwnerWallet"),
+      config.get("delegateOwnerWallet"),
```

Comment:
is this right? shouldn't it be ethOwnerWallet for ethWeb3Config?

Reply:
We want creator-node to be registered with different private keys so that there is no race condition with approve.

ethOwnerWallet is the same for all nodes